src/getUpdatedPackages.js

```diff
@@ -2,15 +2,6 @@ import { execSync } from 'child_process';
 
 export default function getUpdatedPackages(options = {}) {
   const args = options.arguments || [];
-  try {
-    return JSON.parse(
-      execSync(`lerna updated ${args.join(' ')} --json`, { stdio: 'pipe' }).toString()
```

Comment:
Replacing `updated` with `list` here should do the trick!

Reply:
Please also validate whether this works in both lerna 2 and 3. If not, then we need to mark this as a breaking change.